pytests/tests/test_baseurls.py

```diff
@@ -32,11 +32,37 @@ def teardown_test(utils):
         os.remove(filename)
 
 
-def test_multiple_baseurls(utils):
+def test_multiple_repoid(utils):
```

Comment:
Why is this test in `test_baseurls.py`? It has nothing to do with multiple base URLs.

Reply:
Added test case in test_repolist.py